selfdrive/car/ford/carstate.py

```diff
@@ -131,6 +132,10 @@ def get_can_parser(CP):
       messages += [
         ("Lane_Assist_Data3_FD1", 33),
       ]
+    else:
+      messages += [
+        ("INSTRUMENT_PANEL", 10),
+      ]
```

Comment:
Can you check how long this message takes to become valid/present?

Reply:
Bronco Sport
`54827bf84c38b14f|2023-01-26--21-59-07--0`
![image](https://github.com/commaai/openpilot/assets/4038174/87a03e9c-ec83-4b7a-9730-f13fd747e331)

Focus
`e886087f430e7fe7/2024-03-01--00-07-32--0`
![image](https://github.com/commaai/openpilot/assets/4038174/3b4b6f48-92f8-452f-8868-ade0fc4981a8)
`e886087f430e7fe7/2023-07-16--20-52-47--0`
![image](https://github.com/commaai/openpilot/assets/4038174/f128af6f-166c-4b6a-acab-7e6af02637c5)

It looks as though it's probably 0.1-1.0s for us to see the first frame

Reply:
Don't have any rlogs from the metric car

Reply:
Using commaCarSegments
![image](https://github.com/commaai/openpilot/assets/4038174/c8b3d1e3-8795-4292-a888-ec65848fc820)

Reply:
Message is missing on F-150 and Mach-E
![image](https://github.com/commaai/openpilot/assets/4038174/3055b88d-0b03-41b8-9cf9-7d1e4e692bc1)